catalog/NEON/NEON_CHM.jsonnet

```diff
@@ -0,0 +1,169 @@
+
+{
+  stac_version: ee_const.stac_version,
+  type: ee_const.stac_type.collection,
+  stac_extensions: [
+    ee_const.ext_eo,
+    ee_const.ext_sci,
+  ],
+  id: id,
+  title: 'NEON Canopy Height Model',
+  'gee:type': ee_const.gee_type.image_collection,
+  description: ||| 
+	Height of the top of canopy above bare earth (Canopy Height Model (CHM)); 
+	data are mosaicked over the AOP footprint onto a spatially uniform grid 
+	at 1 m spatial resolution. The CHM is generated by creating a continuous 
+	surface of canopy height estimates across the entire spatial domain of 
+	the LiDAR survey, derived from the LiDAR point cloud. The point cloud is 
+	separated into classes representing the ground and vegetation returns. 
+	The ground classified points allow for calculation of a height normalized 
+	point cloud to provide a relative estimate of vegetation elevation. A 
+	surface is then generated using the height normalized vegetation points 
+	to produce the CHM. Any canopy heights less than 2m are set to zero for 
+	data collected with the Optech Gemini LiDAR sensors, and less than 2/3m 
+	for the Riegl and Optech Galaxy sensors. See 
+	[NEON Data Product DP3.30015.001](https://data.neonscience.org/data-products/DP3.30015.001) for more details.
+	Citation: Please use the appropriate citation(s) in your publications; see Terms of Use for more info.
+	Documentation: [NEON DP3.30015.001 Quick Start Guide](https://data.neonscience.org/api/v0/documents/quick-start-guides/NEON.QSG.DP3.30015.001v1?inline=true&fallback=html)
+  |||,
+  license: license.id,
+  links: ee.standardLinks(subdir, id),
+  keywords: [
+    'airborne',
+    'ecology',
+    'lidar',
+    'neon',
+  ],
+  providers: [
+    ee.producer_provider('NEON', 'https://www.neonscience.org/'),
+    ee.host_provider(self_ee_catalog_url),
+  ],
+  extent: ee.extent(-170, 16, -66, 73,'2013-01-01T00:00:00Z',null),
+  summaries: {
+    'gee:schema': [
+      {
+        name: 'AOP_VISIT_NUMBER',
+        description: 'Unique visit number to the NEON site',
+        type: ee_const.var_type.string,
+      },     
+      {
+        name: 'FLIGHT_YEAR',
+        description: 'Year the data were collected',
+        type: ee_const.var_type.string,
+      },  
+      {
+        name: 'NEON_DOMAIN',
+        description: 'NEON eco-climatic domain code, "D01" to "D20". See https://www.neonscience.org/field-sites/about-field-sites',
+        type: ee_const.var_type.string,
+      },
+      {
+        name: 'NEON_SITE',
+        description: 'NEON four-digit site code. See https://www.neonscience.org/field-sites',
+        type: ee_const.var_type.string,
+      },
+      {
+        name: 'NEON_DATA_PROD_ID',
+        description: 'NEON data product identification code, "DP3.30015.001"',
+        type: ee_const.var_type.string,
+      },
+      {
+        name: 'NEON_DATA_PROD_URL',
+        description: 'NEON data product url: "https://data.neonscience.org/data-products/DP3.30015.001"',
```

Comment:
`NEON data product url: "https://data.neonscience.org/data-products/DP3.30015.001"` →

`NEON data product url, e.g. [https://data.neonscience.org/data-products/DP3.30015.001](https://data.neonscience.org/data-products/DP3.30015.001)`

Or is it always this string?

`NEON data product url. Always set to: [https://data.neonscience.org/data-products/DP3.30015.001](https://data.neonscience.org/data-products/DP3.30015.001)`